src/parameters/parameters_artificial_dissipation.h

```diff
@@ -0,0 +1,60 @@
+#ifndef __PARAMETERS_ARTIFICIAL_DISSIPATION_H__
+#define __PARAMETERS_ARTIFICIAL_DISSIPATION_H__
+
+#include <deal.II/base/parameter_handler.h>
+
+namespace PHiLiP {
+namespace Parameters {
+/// Parameters related to the artificial dissipation
+class ArtificialDissipationParam
+{
+public:
+    /// Flag to add artificial dissipation from Persson's shock capturing paper.
+    /** This feature is currently not fully working. It dissipates the Burger's
```

Comment:
Is this comment still accurate?

Reply:
If not, I would suggest removing it

Reply:
I have modified it a bit (it reduces enthalpy for the gaussian bump, but the convergence order is still below (p+1) in the presence of shock).

Reply:
Ok just update it appropriately